packages/react/src/primitives/types/scrollView.ts

```diff
@@ -9,6 +9,12 @@ export interface BaseScrollViewProps extends BaseViewProps {
    * make the width or height of the ScrollView component smaller than the content.
    */
   orientation?: ScrollViewOrientation;
+
+  /**
+   * @description
+   * Scroll to the end as the children of it change
+   */
+  autoScroll?: 'smooth' | 'instant' | 'auto';
```

Comment:
Any potential in the future that we would want to support auto scroll behavior in addition to scrolling to the end of the view?